### ISSUE TITLE
[Bug]: Dying on moon landing. (forge)

### ISSUE THREAD

**Issue by Dycilia** [terrarium-earth/Ad-Astra#175]
### Bug Description

When going to the moon after the loading I stand on the rocket floor speeding towards the moon,
 I immediately die on impact and the rocket is gone. And there is just a small crater at the impact.
 
 (direwolf20 1.19.2 latest version)

### How to Reproduce?

1. Make a rocket and fill with fuel.
2. Launch rocket
3. Pick moon from starmap
4. Load into moon and smack into the ground and die.
5. No rocket or other device at death place, just a crater.

[02Feb2023 15:08:44.551] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Dycilia has reached the goal [One Small Step]
[02Feb2023 15:08:44.556] [Server thread/INFO] [net.blay09.mods.netherportalfix.NetherPortalFix/]: Not storing return portal because it's from ResourceKey[minecraft:dimension / minecraft:overworld] to ResourceKey[minecraft:dimension / ad_astra:moon]
[02Feb2023 15:08:44.558] [Server thread/INFO] [net.blay09.mods.netherportalfix.NetherPortalFix/]: Not storing return portal because it's from ResourceKey[minecraft:dimension / minecraft:overworld] to ResourceKey[minecraft:dimension / ad_astra:moon]
[02Feb2023 15:09:00.873] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Dycilia blew up
[02Feb2023 15:09:00.880] [Server thread/INFO] [com.lothrazar.simpletomb.ModTomb/]: simpletomb.message.new_grave(72, 106, -715) ad_astra:moon

### Expected Behavior

Landing safely on the moon.

### Version

1.12.2

### Mod Loader Version

1.19.2

### Mod Loader

Forge

**Comment by Zarkov2** [comment]
This happened to me as well. Angel ring prevented the fall damage, but the rocket was destroyed.

I didn't find it mentioned in the in-game book (could have missed it), but you need to hold / tap space bar to slow down the descent while landing to avoid a crash. This appears to be the same for all planets.

**Comment by AlexNijjar** [comment]
do you have https://www.curseforge.com/minecraft/mc-mods/minecraft-transit-railway

**Comment by ProsperCraft** [comment]
This is a known bug with https://github.com/jonafanho/Minecraft-Transit-Railway/issues/427  

The same issue presented itself with Beyond earth mod and Space Dimensions mod.

The issue has been open for 8 months.

Since they don't seem to want to fix it on their side(I poked them on their discord as well, said it was a low priority)  Is there any way to force place the player back inside the rocket on dim load?  I see the lander I am on top of it, and it plummets to the moon very quickly and I glide slowly down, I lived by putting on long fall boots right at the end, but the lander/rocket were destroyed.
